docs/_docs/tech/steps.md

```diff
@@ -700,8 +700,10 @@ operator).
 
 ### `formula` step
 
-Add a computation based on other columns or on values.
-Column names must not be escaped. Strings have to be escaped with quotes.
+Add a computation based on a formula. Usually column names do not need to be
+escaped, unless they include whitespaces, in such a case you need to use
+brackets '[]' (e.g. \[myColumn]). Any characters string escaped with quotes
```

Comment:
```suggestion
brackets '[]' (e.g. `[myColumn]`). Any characters string escaped with quotes
```

Reply:
@davinov I think I need to escape the first '[' in markdown to avoid the pattern being considered as a link (my linter reminded me about this :))

Reply:
Also, the markdown seems fine when I preview it in my editor. Tell me if I miss something

Reply:
Yes, it's more to display it in a code block like this: `[myColumn]`
Not a big deal though